File: labs/node/hello/hello.test.js
import { test } from "node:test";
import assert from "node:assert/strict";
import { greet } from "./hello.js";

test("default greeting", () => {
  assert.equal(greet(), "Hello, world!");
});

test("named greeting", () => {
  assert.equal(greet("Claude"), "Hello, Claude!");
});

File: labs/go/hello/hello_test.go
package main

import "testing"

func TestGreet(t *testing.T) {
	cases := map[string]string{
		"":       "Hello, world!",
		"Claude": "Hello, Claude!",
	}
	for in, want := range cases {
		if got := Greet(in); got != want {
			t.Errorf("Greet(%q) = %q, want %q", in, got, want)
		}
	}
}
